README.md

```diff
@@ -120,6 +121,38 @@ job.getQueryResults().on('data', function(row) {});
 ```
 
 
+## Google Cloud Bigtable
+
+- [API Documentation][gcloud-bigtable-docs]
+- [Official Documentation][cloud-bigtable-docs]
+
+*You may need to [create a cluster][gcloud-bigtable-cluster] to use the Google Cloud Bigtable API with your project.*
+
+#### Preview
+
+```js
+var gcloud = require('gcloud');
+
+// Authenticating on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authentication section above).
+var bigtable = gcloud.bigtable({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json',
+  zone: 'my-zone',
+  cluster: 'my-cluster'
+});
+
+var table = bigtable.table('prezzy');
+
+table.getRows(function(err, rows) {});
+
+// Update a row in your table.
+var row = table.row('alincoln');
+
+row.save('follows:gwashington', 1, function(err) {});
```

Comment:
Since Travis isn't going to let us merge anytime soon, is there a way to show an example in this callback for retrieving the row's data to show that it saved correctly?